### PR TITLE
Fix button, improve UX and add GitHub Pages deploy

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js [hunk 1]
```diff
@@ -167,15 +177,43 @@ const Controlled = ({
...
+});
+
+export const CreatorHasSuperLiked = Controlled({
+  isCreator: true,
+  isSuperLikeEnabled: true,
+  hasSuperLiked: true,
+  cooldown: 1,
+});
+
+export const CreatorHadSuperLiked = Controlled({
```
Comment (nnkken) [inline src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js:215]: The names `CreatorHasSuperLiked` and `CreatorHadSuperLiked` are quite confusing
(same for `LikerHasSuperLiked` and `LikerHadSuperLiked`)

Comment (nwingt) [reply src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js:215]: `has` mean the liker has Super Liked, and waiting for CD
`had` mean the liker had Super Liked once, he/she can Super Like again (CD has finished)

Can u suggest another name for the stories?

Comment (nnkken) [reply src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js:215]: `CreatorSuperLikeCountingDown` and `CreatorWithSuperLikeHistory`? But seems too long

Comment (williamchong) [reply src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js:215]: CoolDowning and CooldownReady?

**Review by williamchong — APPROVED** [review]
Should add the built github page url into README for easy access?

File: src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js (hunk 1 above)
Comment (nwingt) [reply src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js:215]: Maybe I add descriptions instead of changing the names later

**Comment by nwingt** [comment]
I'm not admin of the repo, I cannot update GitHub Pages settings

**Comment by williamchong** [comment]
Not sure what's the correct config, added u as repo admin

**Comment by nwingt** [comment]
Maybe merging this PR to deploy g`h-pages` branch first

**Pull request merged**